tests/TorControl/Tests/TorControlTest.php

```diff
@@ -82,4 +82,39 @@ public function testAuthenticate()
 
         $this->assertFalse($this->torControl->getOption('authmethod') === TorControl::AUTH_METHOD_NOT_SET);
     }
+
+    /**
+     * Test multiline replies.
+     */
+    public function testMultilineReplies()
+    {
+        $_cmds = array(
+            'GETINFO version',
+            'GETINFO config-file',
+            'GETINFO config-text', // this should return a "multiline" reply
+            'GETINFO version',
+            'GETINFO config-file'
+        );
+
+        $responses = array();
+
+        $this->torControl->connect();
+        $this->torControl->authenticate();
+
+        foreach ($_cmds as $cmd) {
+            $responses[] = $this->torControl->executeCommand($cmd);
+        }
+
+        // Assert we got the same number of responses entries as commands sent.
+        // If we do, and there are no exceptions, we can handle multi-line replies.
+        $this->assertEquals(count($_cmds), count($responses));
+        // Test to ensure multiline reply codes are correctly populated.
+        foreach ($responses[2] as $resp) {
+            $this->assertSame($resp['code'], '250');
```

Comment:
The first parameter is the expected value and the second the value to test (should be inverted, apply for all other assertions).